Public/Sdk/UnitTests/ValueCache/Test.ValueCache/sealDirectory.lkg

```diff
@@ -22,7 +22,7 @@ Transformer.writeFile(
         items: ["2B"],
     }
 );
-Transformer.sealPartialDirectory(d`./Out/Dir1`, [f`./Out/Dir1/a.txt`]);
+Transformer.sealDirectory({root: d`./Out/Dir1`, files: [f`./Out/Dir1/a.txt`]});

```

Comment:
>sealDirectory [](start = 12, length = 13)

This was a change in the printer where it was always printing partial regardless of the type.